src/backend/distributed/master/master_stage_protocol.c

```diff
@@ -289,11 +289,8 @@ master_append_table_to_shard(PG_FUNCTION_ARGS)
 		}
 	}
 
-	/*
-	 * Abort if all placements failed, mark placements invalid if only some failed. By
-	 * doing this UpdateShardStatistics never works on failed placements.
-	 */
-	CheckForFailedPlacements(true, CoordinatedTransactionUses2PC);
+	/* ensure UpdateShardStatistics never works on failed placements */
```

Comment:
"ensure ... never works on failed" isn't really descriptive of what's going on.  I'd either remove the comment or update it to say something more accurate - but I think the function name is actually plenty descriptive.